app/routes/events/components/StripeElement.js

```diff
@@ -283,7 +278,7 @@ const PaymentRequestForm = (props: PaymentRequestFormProps) => {
     return () => {
       completePaymentManual('fail');
     };
-  }, []);
+  }, [completePaymentManual]);
```

Comment:
@LudvigHz Can I do this? I'm assuming it won't change anything as completePaymentManual is a const, but I don't understand all the Stripe code.

Reply:
The point of this code is just to cancel the payment in case the component unmounts. So if the payment status is suddenly accepted and we need to cancel a started payment, we can do that safely.

As far as I can see, this shouldn't break anything though. But we should test it to make sure.